packages/nextjs/app/builders/_components/Batch4AddressDisplay.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from "next/link";
+import { Batch4Address } from "./Batch4Address";
+import { Batch4Avatar } from "./Batch4Avatar";
+import { useTheme } from "next-themes";
+import { GlobeAltIcon } from "@heroicons/react/24/solid";
+
+const Batch4AddressDisplay = ({ address, index }: { address: string; index: number }) => {
+  const { resolvedTheme } = useTheme();
+  const isDarkMode = resolvedTheme === "dark";
+
+  return (
+    <div className={`flex flex-col items-center justify-center rounded-lg p-5 bg-white shadow-md`}>
+      <div className="mx-auto h-32 w-32 rounded-full overflow-hidden shadow-lg mb-4 relative ">
+        <Batch4Avatar address={address} size={128} />
+      </div>
+      <h3 className={"text-lg font-semibold text-center text-black"}>Builder {index + 1}</h3>
+      <p className={"text-gray-600 text-sm text-center mt-1 text-black"}>
+        <Batch4Address address={address} format="short" size="base" />
+      </p>
```

Comment:
I believe this is causing this error (Should be a div wrapper, not a p)

![image](https://github.com/BuidlGuidl/batch4.buidlguidl.com/assets/2486142/ac92a34d-13dc-47d1-a0d3-5e1f857fa7af)